### PR TITLE
fixed a massive conflicting/hidden files and changes

### PR DESCRIPTION
**Pull request by LinaQh95**

### CODE REVIEW

**Comment by LinaQh95** [comment]
#21 
In general, this is setting up sign up page to make it easier for the user to register to the website and see all their donations details and history.

File: src/App.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import {LandingPage, OrgInfo, Profile, SignUp, DonationPage} from './screens'
 function App () {
     return (
       <div>
+        <SignUp/>
```
Comment (Karyum) [inline src/App.jsx]: delete this line to avoid merge conflicts with other pull requests

File: src/components/SignUpInputs/index.jsx [hunk 2]
```diff
@@ -1,11 +1,290 @@
...
+//                         setPassword(event.target.value)
+//                     }}
+//                 />
+
+//             </div>
+//             <div className="form-control">
+//                 <label htmlFor="fullName">Full name</label>
+
+//                 <input
+//                     className="input-form"
```
Comment (Karyum) [inline src/components/SignUpInputs/index.jsx]: do you need these comments ? if not can we delete them?

File: src/components/SignUpInputs/index.jsx [hunk 3]
```diff
@@ -1,11 +1,290 @@
...
+
+
+
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
```
Comment (Karyum) [inline src/components/SignUpInputs/index.jsx]: too many components here are all of them even used?

File: src/components/SignUpInputs/index.jsx [hunk 4]
```diff
@@ -1,11 +1,290 @@
...
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+export default function SignUpInputs() {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
```
Comment (Karyum) [inline src/components/SignUpInputs/index.jsx]: what does this do?

File: src/components/SignUpInputs/style.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
+
+.form-control {
+    display: flex;
+    flex-direction: column;
+    /* align-content: space-between; */
```
Comment (Karyum) [inline src/components/SignUpInputs/style.css]: remove commented out code in css

File: src/components/SignUpInputs/style.css [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    position: relative;
+    width: 414px;
+    height: 951px;
+    background: #FFFFFF;
+}
+
+
+
+.input-form{
+/* position: absolute; */
```
Comment (Karyum) [inline src/components/SignUpInputs/style.css]: this too and there is much more

File: src/screens/SignUp/style.css [hunk 7]
```diff
@@ -0,0 +1,49 @@
+#signUpHead {
+    /* position: absolute; */
```
Comment (Karyum) [inline src/screens/SignUp/style.css]: again commented out code in this file

**Review by Karyum — CHANGES_REQUESTED** [review]
right now all the text fields are not connected to a state, do you want to do it now ? or later on when you connect it to the API ? (both are the same)

**Review by Karyum — APPROVED** [review]

**Pull request merged**